src/events/app/index.ts

```diff
@@ -241,7 +254,7 @@ async function handleSyncPulls(
 
     // TODO: Optimize the sleep time.
     // In order to avoid frequent access to the API.
-    await sleep(1000);
```

Comment:
If we don't add this sleep, will we reach GitHub's limit on how often the GitHub app can access the api?

Reply:
There was no problem with the half-hour test.

Reply:
![image](https://user-images.githubusercontent.com/5086433/105375167-73c84100-5c43-11eb-9d2d-5d3b97bec79d.png)
hit rate limit after 50min

Reply:
Maybe probot has handle it for us, It will wait for a while and try again.

Reply:
So we can use this mechanism?

Reply:
Yes, but we still need to control the frequency of API access, too fast access will lead to a longer waiting time.